specs/rate-limiting.md

```diff
@@ -0,0 +1,41 @@
+# Rate limiting guidance
+
+Using the rate limiting guidance plugin, developers can ensure that they properly handle rate limiting in their applications.
+
+Rate limiting is an API feature that communicates to API consumers how many resources are left in a given time window. When the API consumer reaches the limit, the API will return a `429 Too Many Requests` response. API consumers can use information about rate limiting to avoid reaching the limit and to build applications that can stay under the throttling limit.
+
+## History
+
+| Version | Date | Comments | Author |
+| ------- | ---- | -------- | ------ |
+| 1.0 | 2023-03-08 | Initial specifications | @waldekmastykarz |
+
+## Configuration
+
+Setting|Description
+-------|-----------
+`headerLimit`|Name of the `RateLimit-Limit` header returned by the API. Default: `RateLimit-Limit`
+`headerRemaining`|Name of the `RateLimit-Remaining` header returned by the API. Default: `RateLimit-Remaining`
+`headerReset`|Name of the `RateLimit-Reset` header returned by the API. Default: `RateLimit-Reset`
+`headerRetryAfter`|Name of the `Retry-After` header returned by the API. Default: `Retry-After`
+`costPerRequest`|Number of resources consumed by a single request. Default: `1`
+`resetTimeWindowSeconds`|Number of seconds before the resource counter is reset. Default: `60`
+`warningThresholdPercent`|Percentage of remaining resources at which the warning should be displayed. Default: `80`
+`rateLimit`|Number of resources available in the time window. Default: `120`
```

Comment:
Where does 120 come from? The lowest value 1200 resource units per minute

Reply:
This is just a random value indeed meant to help folks verify that their code properly handles backing off without having to issue an excessive number of requests first. We can include guidance in our docs to the correct documentation in case folks want to see the behavior closer to Microsoft Graph.